scipy/linalg/flapack_pos_def_tri.pyf.src

```diff
@@ -83,3 +83,34 @@ subroutine <prefix2c>pttrs(lower, n, nrhs, d, e, b, ldb, info)
     integer intent(out) :: info
 
 end subroutine<prefix2c>pttrs
+
+
+subroutine <prefix>pteqr(compute_z, n, d, e, z, ldz, work, info)
+    ! d, e, z, work, info  = pteqr(d, e, z, compute_z=0, overwrite_d=0, overwrite_e=0, overwrite_z=0)
+    ! ?PTEQR computes all eigenvalues and, optionally, eigenvectors of a
+    ! symmetric positive definite tridiagonal matrix.
+    !
+    ! This routine computes the eigenvalues of the positive definite
+    ! tridiagonal matrix to high relative accuracy.  This means that if the
+    ! eigenvalues range over many orders of magnitude in size, then the
+    ! small eigenvalues and corresponding eigenvectors will be computed
+    ! more accurately than, for example, with the standard QR method.
+    !
+    ! The eigenvectors of a full or band positive definite Hermitian matrix
+    ! can also be found if ?HETRD, ?HPTRD, or ?HBTRD has been used to
+    ! reduce this matrix to tridiagonal form.  (The reduction to
+    ! tridiagonal form, however, may preclude the possibility of obtaining
+    ! high relative accuracy in the small eigenvalues of the original
+    ! matrix, if these eigenvalues range over many orders of magnitude.)
+    callstatement (*f2py_func)((compute_z?(compute_z==2?"I":"V"):"N"), &n, d, e, z, &ldz, work, &info)
+    callprotoargument char*, int*, <ctypereal>*, <ctypereal>*, <ctype>*, int*, <ctypereal>*, int*
+    integer intent(in, optional), check((compute_z>=0) && (compute_z<=2)) :: compute_z = 0
+    integer intent(hide), check(n > 1), depend(d) :: n = len(d)
```

Comment:
check is not needed here

Reply:
Same deal with `e` dimension like we discussed in ptsvx wrappers.

Reply:
`dimension((n>0?n-1:0))`

Reply:
Removed check